docs/operator_guides/0_running_an_operator.md

```diff
@@ -212,6 +233,14 @@ journalctl -xfeu aligned-operator.service
 
 To unregister the Aligned operator, run:
 
+- Mainnet:
+
+```bash
+cast send --rpc-url https://ethereum-rpc.publicnode.com --private-key <private_key> 0xA8CC0749b4409c3c47012323E625aEcBA92f64b9 'deregisterOperator(bytes)' 0x00
```

Comment:
Nit: if there is a way to get the private key from stdin it should be the recommended approach: pasting it on the command line risks leaking it to the `.bash_history` in plain text.

Reply:
We can do it in another PR. Cast has interactive mode